x-pack/plugins/observability/server/lib/rules/custom_threshold/custom_threshold_executor.test.ts

```diff
@@ -123,8 +133,7 @@ const setEvaluationResults = (response: Array<Record<string, Evaluation>>) => {
   jest.requireMock('./lib/evaluate_rule').evaluateRule.mockImplementation(() => response);
 };
 
-// FAILING: https://github.com/elastic/kibana/issues/155534
-describe.skip('The metric threshold alert type', () => {
+describe('The metric threshold alert type', () => {
```

Comment:
Unrelated to this PR, but is it easy to to change `threshold alert type` to `threshold rule type`?